Tests/js/views/assets/ez-usermenuitemfireeventview-tests.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuitemfireeventview-tests', function (Y) {
+    var renderTest, eventTest,
+        SELECTOR_MENU_ITEM = '.ez-user-menu-item';
+
+    renderTest = new Y.Test.Case({
+        name: "eZ User Menu Item Fire Event render test",
+
+        setUp: function () {
+            this.view = new Y.eZ.UserMenuItemFireEventView({
+                container: '.container'
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        "Should render the user menu item": function () {
+            var container = this.view.render().get('container');
```

Comment:
usually we test that the template was used and received the correct variables (example https://github.com/ezsystems/PlatformUIBundle/blob/master/Tests/js/views/assets/ez-confirmboxview-tests.js#L26)